compiler/modules/front/expr.bal

```diff
@@ -176,6 +176,22 @@ function codeGenExprForString(ExprContext cx, bir:BasicBlock bb, s:Expr expr) re
     return cx.semanticErr("expected string operand", s:range(expr));
 }
 
+function codeGenArgument(ExprContext cx, bir:BasicBlock bb, s:MethodCallExpr|s:FunctionCallExpr callExpr, bir:FunctionRef func, int i) returns ExprEffect|CodeGenError {
+    s:Expr arg = callExpr.args[i];
+    int n = callExpr is s:FunctionCallExpr ? i : i + 1;
+    if n >= func.signature.paramTypes.length() {
+        if func.symbol == { module: { org: "ballerina", names: ["io"] }, identifier: "println" } {
```

Comment:
Make a `final readonly &` variable (e.g. `IO_PRINTLN_SYMBOL`) for the RHS to avoid constructing it each time